pfcpiface/messages_parse_test.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2025
```

Comment:
@arnavk23, properly update this copyright header. I think you could follo the same approach you follow in the other PRs for new files.